libs/datasets/combined_datasets.py

```diff
@@ -231,6 +231,13 @@ def load_us_timeseries_dataset(
     return MultiRegionDataset.read_from_pointer(pointer)
 
 
+@functools.lru_cache(None)
+def load_test_dataset() -> MultiRegionDataset:
```

Comment:
Suggestion to move this in `tests/test_helpers`. prefer to not have code only used for tests outside of the `tests` folder

Reply:
https://github.com/covid-projections/covid-data-model/pull/1067